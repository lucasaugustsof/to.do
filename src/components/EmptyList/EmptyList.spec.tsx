import { describe, it, expect } from 'vitest'

import { render } from '@testing-library/react'

import { EmptyList } from '.'

describe('Components/EmptyList', () => {
  it('should render the EmptyList component correctly', () => {
    const { asFragment } = render(<EmptyList />)

    expect(asFragment()).toMatchSnapshot()
  })
})
