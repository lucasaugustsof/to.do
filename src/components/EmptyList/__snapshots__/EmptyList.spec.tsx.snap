// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Components/EmptyList > should render the EmptyList component correctly 1`] = `
<DocumentFragment>
  <div
    class="_container_4b25f8"
  >
    <span>
      <strong>
        Você ainda não tem tarefas cadastradas
      </strong>
      <br />
      Crie tarefas e organize seus itens a fazer
    </span>
  </div>
</DocumentFragment>
`;
